From 0927b4c7ca0c5c7676570b21063eb8a092ba30df Mon Sep 17 00:00:00 2001
From: Rodrigo Gryzinski <rogryza@rogryza.com>
Date: Sun, 27 Sep 2020 19:16:59 -0300
Subject: [PATCH 2/2] Fix hetzner

---
 cloudinit/sources/DataSourceHetzner.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cloudinit/sources/DataSourceHetzner.py b/cloudinit/sources/DataSourceHetzner.py
index a86035e0..f5184d15 100644
--- a/cloudinit/sources/DataSourceHetzner.py
+++ b/cloudinit/sources/DataSourceHetzner.py
@@ -100,7 +100,7 @@ class DataSourceHetzner(sources.DataSource):
 
 
 def on_hetzner():
-    return util.read_dmi_data('system-manufacturer') == "Hetzner"
+    return util.read_dmi_data('sys_vendor') == "Hetzner"
 
 
 # Used to match classes to dependencies
-- 
2.28.0

